dev-learning/Models/MyDbContext.cs

```diff
@@ -2,14 +2,15 @@
 using Microsoft.EntityFrameworkCore;
 namespace dev_learning.Models
 {
-    public class UserContext : DbContext
+    public class MyDbContext : DbContext
```

Comment:
`MyDbcontext` is a very generic name. As I've read having different `DbContext` files for different entities `DbSets` depends on the context of those entities - should they be used together(in one controller) at some point? Then they should live under one file. If no - they should live under different files.
Regarding all this, I would suggest naming this something more representative, because there could be one more `DbContext` file.

Reply:
There is no reason to have several different DbContexts for one database. I'm sure we will only use one database, so it's just a matter of giving this DbContext a name which would represent all tables.

Reply:
There is a reason, as I explained for unrelated entities you should have different db contexts